src/util.ts

```diff
@@ -393,3 +393,27 @@ const addGraphTotals = (
   )
   return { totalTxs, totalUsd }
 }
+
+export const movingAveDataSort = (
+  data: Data[]
+): Array<{ date: string; allTxs: number }> => {
+  const aveArray: Array<{ date: string; allTxs: number }> = []
+  if (Array.isArray(data) === false) return aveArray
+  const dataLength = data.length
```

Comment:
This variable does not feel necessary. Only used once inside the `for` loop. I would replace all instances of `dataLength` with `data.length`.

Reply:
Changed according to comment.